tdrs-backend/tdpservice/reports/serializers.py

```diff
@@ -18,6 +19,7 @@ class Meta:
 
         model = ReportFile
         fields = [
+            "id",
```

Comment:
We need the unique `id` for each created ReportFile so that we can link to them when creating corresponding LogEntries.